astropy/io/fits/fitstime.py

```diff
@@ -0,0 +1,284 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import re
+import warnings
+from collections import defaultdict, OrderedDict
+
+import numpy as np
+
+from . import Header, Card
+
+from .column import COLUMN_TIME_KEYWORDS, is_time_column_keyword
+
+from ...coordinates import EarthLocation
+from ...table import Column
+from ...time import Time
+from ...time.core import BARYCENTRIC_SCALES
+from ...time.formats import FITS_DEPRECATED_SCALES
+from ...utils.exceptions import AstropyUserWarning
+
+
```

Comment:
Just leave a comment with link to fits paper for convenience for future developers.